python/pyarrow/tests/parquet/test_encryption.py

```diff
@@ -133,6 +133,32 @@ def test_encrypted_parquet_write_read(tempdir, data_table):
     assert data_table.equals(result_table)
 
 
+def test_uniform_encrypted_parquet_write_read(tempdir, data_table):
+    """Write an encrypted parquet, verify it's encrypted, and then read it."""
+    path = tempdir / PARQUET_NAME
+
+    # Encrypt the footer and all columns with the footer key,
+    encryption_config = pe.EncryptionConfiguration(
+        footer_key=FOOTER_KEY_NAME,
+        uniform_encryption=True,
+        encryption_algorithm="AES_GCM_V1",
+        cache_lifetime=timedelta(minutes=5.0),
+        data_key_length_bits=256)
```

Comment:
Perhaps add
```suggestion
        data_key_length_bits=256)
    assert encryption_config.uniform_encryption is True
```

(and add a similar assert in `test_encrypted_parquet_write_read` above)

Reply:
Done @rok and @pitrou.